Code/GraphMol/Wrap/rough_test.py

```diff
@@ -14,7 +14,7 @@
 from rdkit import DataStructs
 from rdkit import Chem
 import rdkit.Chem.rdDepictor
-from rdkit.Chem import rdqueries
+from rdkit.Chem import rdqueries, rdmolops
```

Comment:
it's not wrong, but you don't technical need `rdmolops` here since it's imported as part of `Chem`

Reply:
Fixed.